applications/esm/README.md

```diff
@@ -0,0 +1,961 @@
+
+# Open-Omics-ESM
+
+Open-Omics-ESM is an optimized version of the Evolutionary Scale Modeling (ESM) toolkit, tailored for modern CPUs. It improves the performance of key ESM modules—such as ESM-embeddings, LM-Design, InverseFolding, and ESMFold—by leveraging Intel Extension for PyTorch (IPEX) and performing computations in lower precision (bf16).
```

Comment:
Are we using IPEX for any application in ESM? If not, we should remove that from this readme file. 

The updated line can be something like this 

Open-Omics-ESM is an optimized version of the Evolutionary Scale Modeling (ESM) toolkit, tailored for modern CPUs. It enhances the performance of key ESM modules—such as ESM-embeddings, LM-Design, InverseFolding, and ESMFold—by leveraging the latest PyTorch and using lower precision (bf16) for computations.

Reply:
I can add the changes for the latest commit to the same PR.